packages/runtime/src/remote/index.ts

```diff
@@ -406,7 +406,10 @@ export class RemoteHandler {
       const remoteInfo = loadedModule.remoteInfo;
       const key = remoteInfo.entryGlobalName as keyof typeof globalThis;
 
-      if (globalThis[key]) {
+      if (
```

Comment:
what happens if its not configurable? we just overwrite it?

Reply:
yes, the remote container will be override since the remoteEntry.js be loaded

Reply:
gotcha